ground/src/main/java/com/google/android/ground/ui/map/gms/GoogleMapsFragment.kt

```diff
@@ -75,19 +71,17 @@ const val MARKER_Z = 3f
  * on window insets.
  */
 @AndroidEntryPoint(SupportMapFragment::class)
-class GoogleMapsFragment : Hilt_GoogleMapsFragment(), Map {
+class GoogleMapsFragment : Hilt_GoogleMapsFragment(), MapView {
+  // TODO: Inject dispatchers.
+  private val defaultDispatcher = Dispatchers.Default
```

Comment:
Just curious why we haven't injected it? Since this class extends `AndroidEntryPoint`, we can simply inject it :)

Reply:
I thought that since we're instantiating it ourselves in `MapProviderModule` that injection wouldn't work. But I guess that since it's an entry point it gets applied after init? I'll give it a try.